public/app/features/alerting/unified/rule-editor/RuleEditorGrafanaRules.test.tsx

```diff
@@ -128,7 +129,7 @@ describe('RuleEditor grafana managed rules', () => {
     const nameInput = await ui.inputs.name.find();
     expect(nameInput).toHaveValue(grafanaRulerRule.grafana_alert.title);
     //check that folder is in the list
-    expect(ui.inputs.folder.get()).toHaveTextContent(new RegExp(folder.title));
+    await waitFor(() => expect(ui.inputs.folder.get()).toHaveTextContent(new RegExp(folder.title)));
```

Comment:
@gillesdemey This test was failing but only when both tests were run. If it's focused then it was passing. I suspect it's related to something being cached between the two. I didn't fully investigate, but it's fixed by waiting for the folder text to be present (think its related to some RTK caching in tests perhaps?)